src/components/MovieCard.js

```diff
@@ -23,19 +24,17 @@ const MovieCard = ({movie, add=false, remove=false}) => {
       };
     return (
         <div className="Cardmovie">
-        <Link to={newTo}>
-        <div className="moviediv">
-            {movie.poster_path ? <img className='movieimg' src={`${image_path}${movie.poster_path}`} alt="" width="100%" height="auto"/> 
-             : null}
-            <h4 className="movietitletext" >{movie.title}</h4>
-        </div>
-        </Link>
+            <Link to={newTo}>
+            <div className="moviediv">
+                {movie.poster_path ? <img className='movieimg' src={`${image_path}${movie.poster_path}`} alt="" width="100%" height="auto"/> 
+                : null}
+                <h4 className="movietitletext" >{movie.title}</h4>
+            </div>
+            </Link>
             <div className="center">
-                 <button className="addButton" hidden={!add} onClick={ handleAdd }>Add</button>
-                 <button className="removeButton" hidden={!remove} onClick={ handleRemove }>Remove</button>
-
+                <button className="addButton" hidden={!add} onClick={ handleAdd }>Add</button>
```

Comment:
This is changed in both this branch and master. id="addButton" reverted. Please sync together on button design